src/fileimpl.cpp

```diff
@@ -601,29 +601,39 @@ std::string pseudoTitle(const Dirent& d)
   return std::string(1, d.getNamespace()) + '/' + d.getTitle();
 }
 
+bool checkIndirectDirentAccessor(const IndirectDirentAccessor& accessor, entry_index_type totalCount) {
```

Comment:
Function names should reflect their semantics rather than their signature, so this function still better be named `checkTitleIndex()` or similar

Reply:
Renamed in `checkTitleListing`